machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc

```diff
@@ -24,6 +24,12 @@ If you do not have access to the {op-system-first} images that were used to crea
 
 Before you add more compute machines to a cluster that you installed on bare metal infrastructure, you must create {op-system} machines for it to use. You can either use an ISO image or network PXE booting to create the machines.
 
+[NOTE]
+====
+It is recommended to use the same Ignition config file and ISO image that you used to install a cluster to deploy all new nodes in a cluster, before or after the cluster upgrades.
```

Comment:
"Recommended" may not be strong enough.  Using the same ISO image is the supported flow, and AFAIK we don't test on clusters installed from mixed bootimages.

Reply:
Agree here. We should use stronger wording.

Reply:
Re-phrased as :
**You must use the same Ignition config file and ISO image that you used to install a cluster to deploy all new nodes in a cluster, before or after the cluster upgrades.**
@bgilbert @travier  what do you think?

Reply:
Added the above .

Reply:
This isn't strictly true of the Ignition config though.  Some users need to supply a different Ignition config for every node, e.g. for configuring a static IP address, and modifying the pointer Ignition config is currently the only way to do that.